packages/@adobe/spectrum-css-temp/components/dropzone/skin.css

```diff
@@ -18,27 +18,32 @@ governing permissions and limitations under the License.
 .spectrum-Dropzone {
   border-color: var(--spectrum-dropzone-border-color);
 
-  &.is-dragged {
+  &[data-focus-visible]{
+    border-color: var(--spectrum-dropzone-border-color-selected-hover);
+  }
+
+  &[data-drop-target] {
+    border-style: solid;
     border-color: var(--spectrum-dropzone-border-color-selected-hover);
     background-color: var(--spectrum-dropzone-background-color-selected-hover);
 
-    .spectrum-IllustratedMessage-illustration {
-      color: var(--spectrum-dropzone-illustratedmessage-illustration-color-selected);
+    &.focus-ring {
+      border-color: var(--spectrum-dropzone-border-color-selected-hover);
     }
   }
+}
 
-  &:focus:not(.is-dragged) {
-    border-color: var(--spectrum-dropzone-border-color);
-    .spectrum-IllustratedMessage-illustration {
+.spectrum-Dropzone--illustratedMessage {
+  
+  &.is-drop-target {
+    svg{
       color: var(--spectrum-dropzone-illustratedmessage-illustration-color-key-focus);
     }
-    &.focus-ring {
-      border-color: var(--spectrum-dropzone-border-color-selected-hover);
-    }
-    &.is-dragged.focus-ring {
-      .spectrum-IllustratedMessage-illustration {
-        color: var(--spectrum-dropzone-illustratedmessage-illustration-color-selected);
-      }
+  }
+
```

Comment:
also chose to remove focus styling (and just keep focus visible) based on other people's opinions so let me know what you think. we can still bring it up to design but wanted to see how other feel